src/scss/_common.scss

```diff
@@ -96,6 +96,10 @@
         height: auto;
```

Comment:
Katex renders the sqrt sign as an svg element with a built-in height
The height of the sqrt sign is 0 pixels if this style is applied